### PR TITLE
Add GirlScript Summer of Code and Global Diversity CFP Day

### PR DESCRIPTION
**Pull request by jancborchardt**
Two very cool related programs, overdue to add them here :)
- https://gssoc.tech
- https://www.globaldiversitycfpday.com

Please review @anubhamane @accakks @jonatoni @sleepypioneer @evalica @alicetragedy  :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,10 +245,10 @@ <h3>RailsGirls Summer of Code</h3>
 					<h3>Outreachy</h3>
 					<p>3-month internships for people traditionally underrepresented in tech. Interns work remotely with mentors from Free and Open Source Software communities.</p>
 				</a>
-				<a class="col-md-4" href="https://www.redhat.com/en/about/women-in-open-source" target="_blank" rel="noopener">
-					<img alt="" src="img/projects/womeninopensource.png" />
-					<h3>Women in Open Source Award</h3>
-					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
```
Comment (evalica) [inline index.html]: Not really sure if you've removed this part intentionally.

File: index.html [hunk 2]
```diff
@@ -285,6 +285,23 @@ <h3>Fund Club</h3>
...
+					<h3>Women in Open Source Award</h3>
+					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
+				</a>
+				<a class="col-md-4" href="https://www.globaldiversitycfpday.com" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/globaldiversitycfpday.jpg" />
+					<h3>Global Diversity CFP Day</h3>
+					<p>Workshops hosted around the globe encouraging and advising newbie speakers to put together your very first talk proposal.</p>
+				</a>
+				<a class="col-md-4" href="https://hacktoberfest.digitalocean.com/" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/hacktober.jpg" />
```
Comment (evalica) [inline index.html]: is (`alt=""`) alts are empty, why put them in the first place? :) we could remove the attribute altogether.

File: index.html [hunk 3]
```diff
@@ -245,10 +245,10 @@ <h3>RailsGirls Summer of Code</h3>
...
 					<p>3-month internships for people traditionally underrepresented in tech. Interns work remotely with mentors from Free and Open Source Software communities.</p>
 				</a>
-				<a class="col-md-4" href="https://www.redhat.com/en/about/women-in-open-source" target="_blank" rel="noopener">
-					<img alt="" src="img/projects/womeninopensource.png" />
-					<h3>Women in Open Source Award</h3>
-					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
+				<a class="col-md-4" href="https://gssoc.tech" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/girlscriptsummerofcode.jpg" />
+					<h3>GirlScript Summer of Code</h3>
+					<p>3 month long Open Source program during summers, with an aim to help beginners get started with Open Source Development while encouraging diversity.</p>
```
Comment (evalica) [inline index.html]: I think it's "during summer", not the plural form.

File: index.html (hunk 1 above)
Comment (jancborchardt) [reply index.html]: It was moved further down, so that all the 3 internship programs are in one row. :)

File: index.html (hunk 2 above)
Comment (jancborchardt) [reply index.html]: It is to make clear they are decorative images, not just forgotten alt tags. :)

See https://a11yproject.com/posts/alt-text/

> Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page. For example, an image providing decoration to an article’s heading could be hidden by giving the image an alt="" or simply alt

File: index.html (hunk 3 above)
Comment (jancborchardt) [reply index.html]: It seems correct cause it’s recurring? (I mostly just copy the text from each project itself)

**Comment by alicetragedy** [comment]
Sorry I was late to the party but I'm on vacation so not checking emails super often. Great addition to the list 💖

**Comment by jancborchardt** [comment]
@alicetragedy no worries and have a nice vacation! :)

**Pull request merged**